docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,419 @@
+
+For more information, see [Naming conventions](audit-log-filter-naming.md)
+
+### audit_log_filter_filter_id
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-file-id |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 1  |
+| Minimum value | 0 |
+| Maximum value | 4292967295 |
+
+Defines the current session's internal ID of the audit log filter file. 
+
+The default value is 0 (zero), which means that the session has no filter assigned.
+
+### audit_log_filter_format
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-format |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NEW  |
+| Available values | OLD, NEW, JSON |
+
+Defines the audit log filter file format. 
+
+The available values are: [OLD (old-style XML)](audit-log-filter-old.md), [NEW (new-style XML)](audit-log-filter-new.md) and [JSON](audit-log-filter-json.md).
+
+### audit_log_filter_format_unix_timestamp
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-format-unix-timestamp |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Boolean  |
+| Default | OFF  |
+
+This option is only supported for JSON-format files.
+
+Enabling this option adds a `time` field to JSON-format files. The `time` field indicates the date and time when the event was generated. Changing the value causes a file rotation because all records must either have or do not have the `time` field. This option requires the `AUDIT_ADMIN` privilege along with the `SYSTEM_VARIABLES_ADMIN` privileges. 
+
+This option does nothing when used with other format types.
+
+### audit_log_filter_handler
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-handler |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | FILE  |
+
+Defines where the plugin writes the audit log filter file. The following values are available:
+
+* `FILE` - plugin writes the log to a location specified in [`audit_log_filter_file`](#audit_log_filter_file)
+* `SYSLOG` - plugin writes to the syslog
+
+
+### audit-log-filter-key-derivation-iterations-count-mean
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-key-derivation-iterations-count-mean |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 60000  |
+| Minimum value | 1000 |
+| Maximum value | 2147483647 |
+
+Defines the mean value of iterations used by the password-based derivation routine used while calculating the encryption key and iv values. The actual iterations count is generated as a random number and deviates no more than 10% from this value.
+
+### audit_log_filter_max_size
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-max-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0 |
```

Comment:
Please change default to 1GB and make sure it is clear it is measured in bytes.

Reply:
What variable is this?